src/http_client.h

```diff
@@ -113,6 +127,8 @@ class http_client
         default_flags = 0
     };
 
+    using hdrs_t = std::vector<std::pair<std::string, std::wstring>>;
```

Comment:
> Let's call it headers, abbreviating too much only hurts readability.

No problem will rename

Because `http_headers::add` expect `wstring` here https://github.com/vslavik/poedit/blob/64075831c26fb0ea91800ebf1213e57c1e047883/src/http_client_casablanca.cpp#L235 while `str::to_NS` in macOS implementation seems accept both (`string` or `wstring`) here https://github.com/vslavik/poedit/blob/64075831c26fb0ea91800ebf1213e57c1e047883/src/http_client_macos.mm#L151 (that is why it could look strange for you if your primary development platform/environment is macOS :) ). Anyway if you feel non comfortable with `wstring` there - I can change it to `string` which would convert to `wstring` here https://github.com/vslavik/poedit/blob/64075831c26fb0ea91800ebf1213e57c1e047883/src/http_client_casablanca.cpp#L241 same way as here https://github.com/vslavik/poedit/blob/64075831c26fb0ea91800ebf1213e57c1e047883/src/http_client_casablanca.cpp#L240

Reply:
> (that is why it could look strange for you if your primary development platform/environment is macOS :) )

That's not it. Since the `http_client` API is narrow-string, it makes sense to keep it that way for headers too.

HTTP headers are not Unicode, but Latin 1, and your conversion from Unicode to ANSI acknowledges it by [explicitly discarding data](https://github.com/vslavik/poedit/blob/64075831c26fb0ea91800ebf1213e57c1e047883/src/http_client_casablanca.cpp#L236). If the public API pretends to accept Unicode strings but then silently damages them, that's bad — using narrow string is explicit acknowledgment of it (hence why http_client API is that way). We don't need Unicode for anything either, so no reason to spent effort on doing it properly (RFC 5987).

Reply:
hdrs_t=>headers and wstring=>string changed 3463dfbc30e05d8deb4e08d9ee0d7d969117cc80